01-JS-DOM-APIs/Ejercicios 1,2,3,4/css/style.css

```diff
@@ -0,0 +1,60 @@
+* {
```

Comment:
Fer, what can I use instead? For example, if I want to set the background-color of all my html content in a particular color...or to set a margin all around the page.

Is better if I set the background-color to all child elements of the body? 

Thanks.